src/utils.py

```diff
@@ -0,0 +1,62 @@
+from os import environ
+import sys
+
+import requests
+
+
+AUTH = None
+AUTH_API_ENDP = environ['AUTH_API_ENDP']
+JWT_DISABLED = environ['JWT_DISABLED'].lower() == 'true'
+URI_API_ENDP = environ['URI_API_ENDP']
+URI_API_USER = environ['URI_API_USER']
+URI_API_PASS = environ['URI_API_PASS']
+URI_SCHEME = environ['URI_SCHEME']
+URI_STRICT = environ['URI_STRICT']
+
+
+def get_token(url, email, passwd):
+    """Fetch token from tokens API."""
+    credentials = {'email': email, 'password': passwd}
+    res = requests.post(url, json=credentials)
+
+    try:
+        assert res.status_code == 200
```

Comment:
Why use `assert`, what's wrong with an `if`??

Reply:
I guess this would be a coding preference - it is an established pattern used in other `hirmeos` drivers so I'd rather not deviate unless I have a good reason.
 
Python does supposedly encourage EAPF: https://docs.python.org/3.7/glossary.html#term-eafp

Also, if the driver is set up correctly, this should never crash, which (I think) makes this _slightly_ faster than `if`.

Reply:
Yeah well, it shows I come from a C style background, I suppose

Reply:
But I don't think it avoids a `compare` statement, which is what `if` is, it'll still end up costing the same

Reply:
And to me it's not as clear as `if`

Sorry these thoughts keep occurring to me

Reply:
Having said all that, if it is the style used throughout `hirmeos`, then stick to it, even if it throws people like me at first.